query/src/main/scala/filodb/query/exec/SelectRawPartitionsExec.scala

```diff
@@ -13,15 +13,31 @@ import filodb.query.Query
 import filodb.query.Query.qLogger
 import filodb.query.exec.rangefn.RangeFunction
 
-object SelectRawPartitionsExec extends  {
+object SelectRawPartitionsExec extends {
+
   import Column.ColumnType._
 
   // Returns Some(colID) the ID of a "max" column if one of given colIDs is a histogram.
   def histMaxColumn(schema: Schema, colIDs: Seq[Types.ColumnId]): Option[Int] = {
     colIDs.find { id => schema.data.columns(id).columnType == HistogramColumn }
-          .flatMap { histColID =>
-            schema.data.columns.find(c => c.name == "max"  && c.columnType == DoubleColumn).map(_.id)
-          }
+      .flatMap { histColID =>
+        schema.data.columns.find(c => c.name == "max" && c.columnType == DoubleColumn).map(_.id)
+      }
+  }
+
+  // Returns Some(colID) the ID of a "min" column if one of given colIDs is a histogram.
+  def histMinColumn(schema: Schema, colIDs: Seq[Types.ColumnId]): Option[Int] = {
+    colIDs.find { id => schema.data.columns(id).columnType == HistogramColumn }
+      .flatMap { histColID =>
+        schema.data.columns.find(c => c.name == "min" && c.columnType == DoubleColumn).map(_.id)
+      }
+  }
+
+  def histMaxMinColumns(schema: Schema, colIDs: Seq[Types.ColumnId]): Option[Int] = {
+    colIDs.find { id => schema.data.columns(id).columnType == HistogramColumn }
+      .flatMap { histColID =>
+        schema.data.columns.find(c => (c.name == "min" || c.name == "max") && c.columnType == DoubleColumn).map(_.id)
```

Comment:
(c.name == "min" || c.name == "max") should this be &&?

Reply:
updated this function to make sure we return just one option[int] value. we use it as a signal to replace the transformers.